language/stdlib/modules/LibraSystem.move

```diff
@@ -394,10 +395,16 @@ module LibraSystem {
         pragma opaque;
         aborts_if false;
         let new_validator_config = ValidatorConfig::spec_get_config(validators[i].addr);
+        // Amazingly, it is able to prove this because get_validator_index_ ensures it
+        // in calling context.
+        requires 0 <= i && i < len(validators);
+        // TODO (dd): Should be able to prove this, but it times out.
+        // Every member of validators is valid.
+        // requires ValidatorConfig::is_valid(validators[i].addr);
```

Comment:
There is a new way how to keep such properties type checked but not verified.  Use `requires [deactivated] P`.